README.md

```diff
@@ -565,7 +569,33 @@ JSON inventory method
       }
     }
   ```
-
+---
+- **Scale Protocols**
+  - If CES Groups is desired `scale_protocols` example below can be used.
+     - For more information about [CES Groups](https://www.ibm.com/support/knowledgecenter/STXKQY_5.0.5/com.ibm.spectrum.scale.v5r05.doc/bl1adm_configcesprotocolservipadd.htm)
```

Comment:
- If [CES groups](https://www.ibm.com/support/knowledgecenter/STXKQY_5.0.5/com.ibm.spectrum.scale.v5r05.doc/bl1adm_configcesprotocolservipadd.htm) are desired then the following example can be used: